### DISCUSSION TITLE
Agent connected with invalid domain/mesh, holding connection.

### DISCUSSION THREAD

**Discussion by prajwalkumarb** [Ylianst/MeshCentral#5796] [Issues] [answered]
MeshCentral HTTP redirection server running on port 80.
MeshCentral v1.1.14, Hybrid (LAN + WAN) mode.
WARNING: Unable to find mongodump.exe, MongoDB database auto-backup will not be performed.
MeshCentral Intel(R) AMT server running on XXXXXXXXX:4433.
MeshCentral HTTPS relay server running on XX.XX.XX.XX:453.
MeshCentral HTTPS server running on XX.XX.XX.XX:443.
Agent connected with invalid domain/mesh, holding connection (106.51.74.251:63919, mesh//Bh6IZ749e06JQk5c9HgCqG1YNDenJ6wX05fvd5$rfz9M9DAZycbwQ8uTCNQ8V7CY).

What kind of error is this how to solve

**Comment by si458** [comment, marked as answer]
Basically it's saying a computer tried to connect to ur meshcentral, however the domain name in its .msh file or the meshid for the group is invalid and not recognised. So u might need to find out which computer it is and then reinstall the meshagent
